doc/refman/RefMan-tac.tex

```diff
@@ -2688,6 +2703,68 @@ \subsection{\tt inversion \ident}
 Note that \texttt{H} has been substituted by \texttt{(LeS n m0 l)} and
 \texttt{m} by \texttt{(S m0)}.
 
+\example{Using \texorpdfstring{\texttt{inversion\_sigma}}{inversion\_sigma}}
+
+Let us consider the following inductive type of length-indexed lists,
+and a lemma about inverting equality of \texttt{cons}:
+
+\begin{coq_eval}
+Reset Initial.
+Require Coq.Logic.Eqdep_dec.
+\end{coq_eval}
+
+\begin{coq_example*}
+Inductive vec A : nat -> Type :=
+| nil : vec A O
+| cons {n} (x : A) (xs : vec A n) : vec A (S n).
+
+Lemma invert_cons : forall A n x xs y ys,
+    @cons A n x xs = @cons A n y ys
+    -> xs = ys.
+Proof.
+intros A n x xs y ys H.
+\end{coq_example*}
+
+\begin{coq_example}
+Show.
+\end{coq_example}
+
+After performing \texttt{inversion}, we are left with an equality of
+\texttt{existT}s:
+
+\begin{coq_example}
+inversion H.
+\end{coq_example}
+
+We can turn this equality into a usable form with
+\texttt{inversion\_sigma}:
+
+\begin{coq_example}
+inversion_sigma.
+\end{coq_example}
+
+To finish cleaning up the proof, we will need to use the fact that
+that all proofs of \texttt{n = n} for \texttt{n} a \texttt{nat} are
+\texttt{eq\_refl}:
+
+\begin{coq_example*}
+  match goal with
+  | [ H : ?x = ?x :> nat |- _ ] => pose proof (Eqdep_dec.UIP_refl_nat H); subst H
+  end.
```

Comment:
First of all, cannot you do the same thing in a less convoluted way. In particular, you are using non-linear pattern-matching. Isn't it something that is not well supported and we shouldn't be encouraging people to use?
Second, you started to indent the proof script at this point only.
Third, the line overflows when building the PDF.
Last but not least, this line just emits an error and you would see it if it wasn't in a block that doesn't print the result.